hassio/src/system/hassio-supervisor-info.ts

```diff
@@ -55,6 +60,23 @@ class HassioSupervisorInfo extends LitElement {
                 : ""}
             </tbody>
           </table>
+          <div class="options">
+            <ha-settings-row>
+              <span slot="heading">
+                Share Diagnostics
```

Comment:
Let's make it clear we only share when things go wrong
```suggestion
                Share Crash Reports
```

Reply:
Or is this so we can increase scope in the future without asking again?

Reply:
Think this title "Share Diagnostics" is cleaner and we specify it on the next line with Crash Reports. I think it's more open in that way. In the end, we want only to see unknown issues which we can fix and no statistics or other things which are part of the updater on the core.